build.sh

```diff
@@ -220,6 +220,22 @@ case "$COMMAND" in
 esac
 export CONFIGURATION
 
+: ${REALM_SWIFT_VERSION:=1.2}
+case "$REALM_SWIFT_VERSION" in
+    "1.2")
+        DEVELOPER_DIR="/Applications/Xcode.app/Contents/Developer/"
+        REALM_SWIFT_VERSION=
+        ;;
+    "2.0")
+        DEVELOPER_DIR="/Applications/Xcode-beta.app/Contents/Developer/"
```

Comment:
We talked on Slack about the problems with hard-coding paths to Xcode here. We'll need to revisit this when Xcode 7 GMs.